CMakeLists.txt

```diff
@@ -799,6 +814,7 @@ function (config_with_llvm)
     DEPENDS prepare_cmds)
 endfunction()
 
+execute_perf_protobuf()
 if (DEFINED WITH_LLVM)
```

Comment:
I thought the intent was to remove the WITH_LLVM flag and make it unconditional to build all the tools? Did you revert these changes to support the gcov build?

Reply:
This flag still needs to be there, because some source files need to be compiled both with LLVM and without LLVM, so it is not easy to merge the build of "WITH_LLVM" and "WITHOUT_LLVM". We still to do two configs to build two sets of binaries, one for gcc (gcov, etc), the other for create_llvm_prof.

Reply:
Ok, then lets make the variable setting more explicit. Can you add something which can only be set to GCOV or LLVM (defaults to LLVM) since that's our primary use case? Also add a print statement during the cmake step to indicate to the user which type of profile tools will be built.

Reply:
Done.